src/QsCompiler/TestProjects/test12/test12.csproj

```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.Quantum.Sdk/0.24.217480-beta">
```

Comment:
Normally we don't update those projects, but in this case, this beta build is very close to the current one, so it's a good opportunity to migrate to a regular release build for this test.